services/translations.ts

```diff
@@ -390,10 +390,6 @@ export const translations = {
     english: 'You must agree to the Terms and Conditions',
     spanish: 'Debe aceptar los Términos y Condiciones'
   },
-  termsVersion: {
-    english: 'Terms Version 1.0',
-    spanish: 'Versión de Términos 1.0'
-  },
```

Comment:
Here is where the version is being set but it's now hard coded above - we should be able to dynamically set this, no?